parse_document.py

```diff
@@ -14,7 +14,9 @@ def setup_logger(args):
         "error": logging.ERROR
     }
     logger.setLevel(log_levels.get(args.log_level, logging.INFO))
-    file_handler = logging.FileHandler(args.report_file, "w")
+    if not os.path.exists("logs"):
+        os.makedirs("logs", exist_ok=True)
+    file_handler = logging.FileHandler(f"logs/{args.report_file}", "w")
```

Comment:
Actually, I don't think we should be doing this at all (hardcoding to a logs dir). If the user wants to use a specific directory, it can be specified using the path in the report_file arg.

Reply:
Updated the code to not have the log path hardcoded and relied on os.path.join instead